CHANGELOG.adoc

```diff
@@ -3,6 +3,10 @@ All wesentlichen Änderungen für dieses Projekt werden hier dokumentiert.
 
 Das Format basiert auf link:https://keepachangelog.com/en/1.0.0[Keep a Changelog].
 
+## [12.34.3] - 2022-??-??
+
+Fehlermeldungen können nun auf zwei verschiedene Arten verarbeitet und dargestellt werden.
```

Comment:
In Zukunft die Issue-Number angeben und es etwas detailierter beschreiben
(man kann auch auf die Doku verweisen).
In diesem Fall bspw.: Fehlermeldungen von SQL-Prozeduren so ausgeben,
dass Clients diese übersetzen können.